tests/foreman/ui/test_contenthost.py

```diff
@@ -129,6 +129,16 @@ def set_ignore_facts_for_os(value=False):
     ignore_setting.update({'value'})
 
 
+@pytest.fixture(scope='module')
+def set_remote_execution_by_default(value=True):
```

Comment:
Fixtures should never take args/kwargs that have a default value, as all of the parameters of the fixture are resolved by other fixtures.

In order to implement the behavior I was talking about in the other thread, you would take `request` as a kwarg here (no default), and then reference `request.param` to get the value of the indirect parametrization.

You can just default to `True` if requests.param is empty (no parametrization of the fixture indirectly).

Then the tests would be decorated like this if you want to try both settings: `pytest.mark.parametrize('set_remote_execution_by_default', [True, False], indirect=true)`.

Reply:
Thanks for that suggestion @mshriver , I have updated the fixture. In this case I do not want to parametrise the tests. When the test is passing I can move the fixture to `pytest_fixtures/core/`.

Reply:
Moved it